src/k8s-extension/azext_k8s_extension/partner_extensions/OpenServiceMesh.py

```diff
@@ -52,8 +52,8 @@ def Create(self, cmd, client, resource_group_name, cluster_name, name, cluster_t
         # NOTE-2: Return a valid Extension object, Instance name and flag for Identity
         create_identity = True
 
-        if cluster_type == "connectedClusters":
-            _validate_tested_distro(cmd, resource_group_name, cluster_name, version, release_train)
+        # if cluster_type == "connectedClusters":

```

Comment:
If you don't need this validation anymore, remove the code altogether, not leave it commented out.  If you need it later, you will still be able to recover from this commit.